jdaviz/configs/imviz/plugins/orientation/orientation.vue

```diff
@@ -0,0 +1,186 @@
+<template>
+  <j-tray-plugin
+    description="Rotate the viewer orientation or choose to align images by pixels."

```

Comment:
```suggestion
    description="Rotate the viewer orientation or choose to align images by WCS."
```
The phrasing implies WCS is default, and switch to pixel is the functionality. Reverting for this PR (which can be switched again if/when we change to WCS by default in another PR).

Reply:
Good point, but I think this should be completely rewritten if we're reverting it for now (since you can't rotate when pixel-linked), sigh.

Reply:
how about "... or choose to switch between aligning images by WCS (Sky) or pixels." so then we don't need to remember to change it later?

Reply:
Sure!